packages/holocron/src/index.js

```diff
@@ -49,4 +50,5 @@ export {
   getLoadingPromise,
   RenderModule,
   holocronModule,
+  forceLoadModule,
```

Comment:
Might it make more sense for us to do a deep import rather than adding to the public API?

Reply:
Because this needs to be available at runtime in the browser in the global scope I don't know if we can do a deep import?

The usage is essentially `globalThis.Holocron.forceLoadModule(...)` within one app.

Other use cases (such as the example one) can actually import it so deep importing could work.